lib/puma/server.rb

```diff
@@ -321,6 +321,8 @@ def handle_servers
           remote_addr_value = @options[:remote_address_value]
         when :header
           remote_addr_header = @options[:remote_address_header]
+        when :proxy_protocol
+          remote_addr_proxy_protocol = @options[:remote_address_proxy_protocol]
```

Comment:
Do we need to define `remote_addr_proxy_protocol` outside of the case block? 

That's done for `remote_addr_value`, `remote_addr_header` (lines 316, 317)

Reply:
Does not seem like we need to:

```
$ ruby -e 'case 1;when 2;foo="foo";end;p foo'
nil
```

(I think lines 316, 317 can be removed, but that could/should be done in another PR)